packages/salesforcedx-vscode-soql/src/client/codeCompletion.ts

```diff
@@ -142,34 +136,49 @@ const expandFunctions: {
       return [];
     }
 
-    const sobjectFields = objMetadata.fields.reduce((fieldItems, field) => {
-      fieldItems.push(
-        newCompletionItem(
-          field.name,
-          field.name,
-          CompletionItemKind.Field,
-          field.type
-        )
-      );
-      if (field.relationshipName) {
+    const sobjectFields = objMetadata.fields
+      .filter(field => objectFieldMatchesSOQLContext(field, soqlContext))
```

Comment:
Can we modify or add test cases to cover some of these uncovered lines?

Reply:
Thanks Brian. Those lines are covered by integration tests. Seems like some of CodeCov reports do not take the integration tests into consideration.

I've just added more tests around GROUP BY / ORDER BY, covering the `isPreferredItem` case below.